Datahub.Web/Data/DynamodbService.cs

```diff
@@ -0,0 +1,61 @@
+using System;
+using System.Threading.Tasks;
+using Amazon;
+using Amazon.DynamoDBv2;
+using Amazon.DynamoDBv2.DocumentModel;
+using Amazon.Runtime;
+using Datahub.Web.Models;
+using Datahub.Web.Pages.Helpers;
+using Newtonsoft.Json;
+
+namespace Datahub.Web.Data 
+{
+    public interface IDynamodbService
+    {
+        AmazonDynamoDBClient Client();
```

Comment:
No need for this is there? I don't think we need to expose the AWS client at all. The ElasticsearchService does but it's less encapsulated.

Reply:
(I'm talking about `AmazonDynamoDBClient Client()` btw.)

Reply:
Resolved 2bb83f7a3ec5ba4f8e9d9543746a6fcec51edb5b